tests/vm

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+set -eux
+
+# Install LXD
+install_lxd
+
+# Configure LXD
+lxd init --auto
+
+IMAGE="${TEST_IMG:-ubuntu-minimal-daily:24.04}"
+vmName="test-vm"
+
+# Launch test instance
+lxc launch "${IMAGE}" "${vmName}" --vm
```

Comment:
I'd go with `v1` hardcoded everywhere to make it easier to read through (when it grows longer). This also better fit with other test scripts IMHO.